bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsStrategy.scala

```diff
@@ -50,6 +62,11 @@ trait AgsStrategy {
    * Indicates the bands that will be used for a given probe
    */
   def probeBands: BandsList
+
+  /**
+    * Determine if guide speed is applicable to this strategy.
+    */
+  def hasGuideSpeed: Boolean = true
```

Comment:
This one bugs me a bit.  It isn't that an AGS strategy has guide speed options or not.  I think guide speed isn't a factor in NGS2 because the CWFS "guiders" are really guide windows on a detector and the PWFS1 "guide star" isn't used for guiding but rather for slow focus sensing.